packages/async-operations/tests/test-async_operations.js

```diff
@@ -88,8 +88,6 @@ test.before(async (t) => {
     promise: () => Promise.resolve(t.context.functionConfig),
   });
 
-  t.context.createSpy = sinon.spy((record) => Promise.resolve(record));
-  t.context.deleteSpy = sinon.spy(() => true);
   t.context.stubbedAsyncOperationsModel = class {
```

Comment:
This one also seems unnecessary, we are passing it as a param in a number of calls still but I'm not sure it's even listed in the params for `createAsyncOperation` now.

Reply:
411a068b65